recipes/approvaltests.cpp/all/test_package/CMakeLists.txt

```diff
@@ -8,23 +8,25 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 CONAN_BASIC_SETUP()
 
+find_package(ApprovalTests REQUIRED CONFIG)
+
 if(WITH_BOOSTTEST)
     ADD_EXECUTABLE(test_package_boosttest test_boosttest.cpp)
-    target_link_libraries(test_package_boosttest ${CONAN_LIBS})
+    target_link_libraries(test_package_boosttest ApprovalTests::ApprovalTests Boost::Boost)
```

Comment:
Is it mandatory to link explicitly with `Boost::Boost` ? isn't boost automatically part of the link as being a requirement of `approvaltests.cpp` package ?
dito for the 4 next changes

Reply:
Thanks. When I made the changes and tested them on my machine, I could have sworn that I got missing symbols for all the 3rd-part libraries except in Boost...

However, I just repeated that check, and *didn't* get the missing symbols - and so I've just pushed a new change, with only the `ApprovalTests::ApprovalTests` stated...